charts/tempo-distributed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 0.27.19
+version: 0.27.20
```

Feature added should be bumped to minor version.
```suggestion
version: 0.28.0
```